itl/command-plugins.conf

```diff
@@ -1696,6 +1696,7 @@ object CheckCommand "by_ssh" {
 	}
 
 	vars.by_ssh_address = "$check_address$"
+	vars.by_ssh_port = "$ssh_port$"
```

Comment:
I guess you are using apply rules for the services, so you could just solve it like this:

```
apply Service "by-ssh..." {
  if (host.vars.ssh_port) {
    vars.by_ssh_port = host.vars.ssh_port
  }
  ...
}
```